lib/Models/CkanCatalogGroup.ts

```diff
@@ -84,9 +85,13 @@ export class CkanServerStratum extends LoadableStratum(CkanCatalogGroupTraits) {
         .segment("api/3/action/package_search")
         .addQuery({ start: 0, rows: 1000, sort: "metadata_created asc" });
 
-      Object.keys(filterQuery).forEach((key: string) =>
-        uri.addQuery(key, filterQuery[key])
-      );
+      if (typeof filterQuery === "string") {
+        uri.query(uri.query() + "&" + filterQuery.toString());
```

Comment:
Looks good @mwu2018 
Sorry to nitpick, but is it possible to use `uri.addQuery()` instead of `uri.query(uri.query() + ...`?

Reply:
@nf-s You did not nitpick at all. Instead, you have opened a can of worms 😃 
1. The `filterQuery` trait is changed similar to what you suggested initially. Your later suggestion does not allow mixed elements in the array. 
2. We can not use `uri.addQuery(filterQuery)` because the URI encoder will escape `=`, which is not what we want. I have added some comments in the code.
3. The code is slightly refactored in order to do simple unit tests.

Can you please take another look?